*** ./smali/android/provider/Settings$System.smali	2016-01-04 20:39:14.085237399 +0800
--- /home/ghoul/micode/m8/temp/new_smali/framework.jar.out/./smali/android/provider/Settings$System.smali	2016-01-04 20:38:08.486113203 +0800
***************
*** 1527,1532 ****
--- 1527,1534 ----
      iput v0, p1, Landroid/content/res/Configuration;->fontScale:F
  
      :cond_0
+     invoke-static {p0, p1, p2}, Landroid/provider/MiuiSettings;->getConfigurationForUser(Landroid/content/ContentResolver;Landroid/content/res/Configuration;I)V
+ 
      return-void
  .end method
  
