*** ./smali/com/android/internal/telephony/Phone.smali	2016-01-04 20:39:30.721009619 +0800
--- /home/ghoul/micode/m8/temp/new_smali/telephony-common.jar.out/./smali/com/android/internal/telephony/Phone.smali	2016-01-04 20:38:24.897897587 +0800
***************
*** 376,381 ****
--- 376,384 ----
  .method public abstract getMessageWaitingIndicator()Z
  .end method
  
+ .method public abstract getMiuiIccPhoneBookInterfaceManager()Lcom/android/internal/telephony/MiuiIccPhoneBookInterfaceManager;
+ .end method
+ 
  .method public abstract getMsisdn()Ljava/lang/String;
  .end method
  
