*** ./smali/android/app/LoadedApk.smali	2016-01-04 20:39:17.217194683 +0800
--- /home/ghoul/micode/m8/temp/new_smali/framework.jar.out/./smali/android/app/LoadedApk.smali	2016-01-04 20:38:11.638071980 +0800
***************
*** 1812,1817 ****
--- 1812,1823 ----
  
      iput-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
  
+     iget-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
+ 
+     iget-object v1, p0, Landroid/app/LoadedApk;->mPackageName:Ljava/lang/String;
+ 
+     invoke-static {v0, v1}, Landroid/miui/ResourcesManager;->initMiuiResource(Landroid/content/res/Resources;Ljava/lang/String;)V
+ 
      :cond_0
      iget-object v0, p0, Landroid/app/LoadedApk;->mResources:Landroid/content/res/Resources;
  
