*** ./smali/android/net/IConnectivityManager.smali	2016-01-04 20:39:22.881117238 +0800
--- /home/ghoul/micode/m8/temp/new_smali/framework.jar.out/./smali/android/net/IConnectivityManager.smali	2016-01-04 20:38:17.209998890 +0800
***************
*** 447,456 ****
--- 447,472 ----
      .end annotation
  .end method
  
+ .method public abstract startUsingNetworkFeatureMSim(ILjava/lang/String;Landroid/os/IBinder;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract stopUsingNetworkFeature(ILjava/lang/String;)I
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract stopUsingNetworkFeatureMSim(ILjava/lang/String;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
