*** ./smali/android/net/wifi/SupplicantStateTracker$DefaultState.smali	2016-01-04 20:39:22.505122385 +0800
--- /home/ghoul/micode/m8/temp/new_smali/framework.jar.out/./smali/android/net/wifi/SupplicantStateTracker$DefaultState.smali	2016-01-04 20:38:16.822003989 +0800
***************
*** 151,156 ****
--- 151,160 ----
      # setter for: Landroid/net/wifi/SupplicantStateTracker;->mAssociationRejectCount:I
      invoke-static {v2, v4}, Landroid/net/wifi/SupplicantStateTracker;->access$702(Landroid/net/wifi/SupplicantStateTracker;I)I
  
+     iget v2, p1, Landroid/os/Message;->arg1:I
+ 
+     invoke-static {v2}, Landroid/net/wifi/SupplicantStateTrackerInjector;->handleConnectNetwork(I)V
+ 
      goto :goto_0
  
      :sswitch_4
